### PR TITLE
Add Event Tracing use case

### PR DESCRIPTION
**Pull request by lfourie**
Signed-off-by: louis.fourie@huawei.com

### CODE REVIEW

**Comment by duglin** [comment]
@lfourie I believe the format of the "sign off" is incorrect. Please make sure you use "-s" on your git commit - that'll add it in the right format (   name \<email\> )

File: use-cases.md [hunk 1]
```diff
@@ -79,6 +79,11 @@ transited and allow for better introspection of the data.
 
 ### Event Tracing
 
+An event sent from a source may result in a sequence of additional events
+sent from various middleware devices such as event brokers and gateways.
+CloudEvents adds metadata to events to associate these events as being part
+of an event sequence for the purpose of event tracing and troubleshooting.
```
Comment (maplebed) [inline use-cases.md]: Is this going to get a follow-on PR describing what specifically should be added to the metadata to facilitate the trace?

**Comment by duglin** [comment]
@lfourie can you please sign your commit

File: use-cases.md (hunk 1 above)
Comment (lfourie) [reply use-cases.md]: there will be a follow-on PR

**Pull request closed without merging**